includes/class-wcpdf-main.php

```diff
@@ -588,6 +590,50 @@ public function init_tmp () {
 		}
 	}
 
+	public function load_custom_and_local_fonts() {
+		require WPO_WCPDF()->plugin_path() . '/vendor/autoload.php';
+
+		$options = array(
+			'defaultFont'				=> 'dejavu sans',
+			'tempDir'					=> $this->get_tmp_path( 'dompdf' ),
+			'logOutputFile'				=> $this->get_tmp_path( 'dompdf' ) . "/log.htm",
+			'fontDir'					=> $this->get_custom_fonts_path(),
+			'fontCache'					=> $this->get_custom_fonts_path(),
+			'isRemoteEnabled'			=> true,
+			'isFontSubsettingEnabled'	=> true,
+			'isHtml5ParserEnabled'		=> true,
+		);
+
+		if ( ! @is_dir( $this->get_dompdf_fonts_path() ) || ! wp_is_writable( $this->get_dompdf_fonts_path() ) ) {
+			$rootDir = $this->get_custom_fonts_path();
+		}
+
+		// save custom fonts
+		$custom_fonts_options = new Options( $options );
+		if( isset( $rootDir ) ) {
+			$custom_fonts_options->setRootDir( $rootDir );
+		}
+		$dompdf = new Dompdf( $custom_fonts_options );
+		$dompdf->getFontMetrics()->saveFontFamilies();
+
+		// save local fonts
```

Comment:
I'm not sure why this step is necessary. dompdf should merge the dist fonts (from `$rootDir`) with the local fonts (from `$fontDir`) automatically when it's creating the PDF, no?